js/tests/unit/tooltip.js

```diff
@@ -1180,4 +1180,39 @@ $(function () {
     assert.strictEqual($tooltip.data('bs.tooltip'), undefined, 'should not initialize the tooltip')
   })
 
+  QUnit.test('should not remove tooltip if multiple triggers are set and one is still active', function (assert) {
+    assert.expect(41)
+    var $el = $('<button/>').bootstrapTooltip({ trigger: 'click hover focus', animation: false })
```

Comment:
* Use proper HTML: `$('<button>Trigger</button>')`
* Add `.appendTo('#qunit-fixture')` like in the other tests